server-docs/src/test/java/org/neo4j/doc/server/rest/security/CommunityServerTestBase.java

```diff
@@ -78,6 +89,33 @@ protected String txCommitURL()
         return databaseURL() + "tx/commit";
     }
 
+    protected String txCommitURL( String database )
+    {
+        return server.baseUri().resolve( txCommitEndpoint( database ) ).toString();
```

Comment:
Could've reused the `databaseURL` method by having one version where it takes a database name